doc_source/launch-workers.md

```diff
@@ -148,6 +148,15 @@ If you do not provide a keypair here, the AWS CloudFormation stack creation fail
       ```
 
    1. Open the file with your favorite text editor\. Replace the *<ARN of instance role \(not instance profile\)>* snippet with the **NodeInstanceRole** value that you recorded in the previous procedure, and save the file\.
+
```

Comment:
It looks like this section accidentally slipped in from #68. Can you remove this section from this PR or close the other one?

Reply:
I closed #68.